module/config/actor.mjs

```diff
@@ -137,6 +137,22 @@ export const TRAVEL_PACES = freezeEnum({
   }
 });
 
+/**
+ * Categories a language can (optionally) belong to
+ * @type {Record<string, string>}
+ */
+export const LANGUAGE_CATEGORIES = {};
+
+/**
+ * Languages a creature can know
+ * @type {Record<string, {label: string, category?: string}>}}
+ */
+export const LANGUAGES = {
+  common: {
```

Comment:
A nice idea which could be pretty setting agnostic would be to also add `sign` as a language under a category for non-spoken languages.